src/components/SessionExpired.js

```diff
@@ -0,0 +1,46 @@
+import React, {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Card from 'components/Card';
+import ErrorMessage from 'components/Errors/ErrorMessage';
+import Link from 'components/Link';
+
+import {ConfigContext} from '../Context';
+import {flagNoActiveSubmission} from '../data/submissions';
+import useSessionTimeout from '../hooks/useSessionTimeout';
+
+const SessionExpired = () => {
+  const config = useContext(ConfigContext);
+  const [expired] = useSessionTimeout();
+
+  if (!config.debug && !expired) {
+    throw new Error("The session isn't expired");
+  }
+
+  flagNoActiveSubmission();
+  window.localStorage.clear();
+  window.sessionStorage.clear();
+
+  return (
+    <Card
+      title={
+        <FormattedMessage
+          description="Session expired card title"
+          defaultMessage="Your session has expired"
+        />
```

Comment:
To make sure all translations are known, please also run `yarn makemessages` and `yarn compilemessages`

If these exact messages already existed, this will be a no-op so it's safe to always run.